src/catalog/src/builtin/notice.rs

```diff
@@ -95,8 +97,8 @@ pub static MZ_NOTICES_REDACTED: BuiltinView = BuiltinView {
 FROM
     mz_internal.mz_notices
 ",
-    sensitivity: DataSensitivity::SuperuserAndSupport,
-};
+    access: vec![SUPPORT_SELECT, MONITOR_REDACTED],
```

Comment:
Missing a role here?
```suggestion
    access: vec![SUPPORT_SELECT, MONITOR_REDACTED, MONITOR],
```